package.json

```diff
@@ -110,7 +110,7 @@
     "ipfs-block": "~0.7.1",
     "ipfs-block-service": "~0.14.0",
     "ipfs-multipart": "~0.1.0",
-    "ipfs-repo": "~0.20.0",
+    "ipfs-repo": "~0.22.0",
```

Comment:
@jacobheun CI is not happy with the latest ipfs-repo. How did you run the tests before (ref: https://github.com/ipfs/js-ipfs-repo/pull/168#issuecomment-392808793)

Reply:
This is due to the ipld PR, https://github.com/ipld/js-ipld/pull/129, that needs released. I will update https://github.com/ipfs/js-ipfs/pull/1358 once that is done.

The current versioning scheme results in old leveldown being installed for ipld, which breaks the  install.

Reply:
@diasdavid I already have a PR open for node 10 that is running tests now, https://github.com/ipfs/js-ipfs/pull/1358. It also includes the fix for the breaking change in ipfs-repo 0.20.1 https://github.com/ipfs/js-ipfs-repo/issues/165, which is the errors your PR is getting.

Reply:
Missed that one. I had read from your message that you had tested against js-ipfs master. All good then :)